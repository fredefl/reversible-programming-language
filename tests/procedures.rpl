read test;
test += 3;
print test;

read number;

call div;
print output;

proc div
output += number / 5;
end;

number += 10;
call div;
print output;
print number;