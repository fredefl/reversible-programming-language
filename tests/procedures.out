8
2
4
20